code/modules/surgery/organs/lungs.dm

```diff
@@ -371,6 +371,14 @@
 
 		handle_breath_temperature(breath, H)
 		breath.garbage_collect()
+
+	// Hyper-Nob
+		gas_breathed = breath_gases[/datum/gas/hypernoblium][MOLES]
+		if (gas_breathed > gas_stimulation_min)
+			var/existing = H.reagents.get_reagent_amount(/datum/gas/hypernoblium)
+			H.reagents.add_reagent(/datum/reagent/hypernoblium,max(0, 1 - existing))
+		breath_gases[/datum/gas/hypernoblium][MOLES]-=gas_breathed
```

Comment:
Can't remember, but what's the proportion of breath_gasses to the gasses around you?

Reply:
Its copy from Stimulum which I assume would be a balanced number.

Reply:
aight that works